src/gui/mainwindow.h

```diff
@@ -103,7 +103,7 @@ class MainWindow: public QMainWindow
     void showNotificationBaloon(QString title, QString msg) const;
 
 private slots:
-    void toggleVisibility(QSystemTrayIcon::ActivationReason e = QSystemTrayIcon::Trigger);
+    void toggleVisibility(QSystemTrayIcon::ActivationReason reason = QSystemTrayIcon::Trigger);
```

Comment:
>to clear this up, are there any rules you have in mind?

Don't pass parameters of POD types as const reference, since it doesn't have any sense.
Declaration and definition signatures should be the same.
You can use const parameters if it doesn't lead to creating extra variable in function.